.changeset/wicked-poets-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `declaration-property-value-keyword-no-deprecated` autofix for `border-color`
```

Comment:
I don't know how to convey that we replaced a buggy autofixing of `border-color` due to a typo (#8328) by just reporting without `Removed`.

```suggestion
Removed: `declaration-property-value-keyword-no-deprecated` autofix for `border-color`
```

This implies it was working and the former implies we still have autofix for `border-color`.

@jeddy3 what do you think?

Reply:
```suggestion
Removed: `declaration-property-value-keyword-no-deprecated` faulty autofix for `border-color`
```